bootstrap/src/init.py

```diff
@@ -0,0 +1,49 @@
+"""Initialization for the bootstrap server"""
+import inspect
+import logging
+
+import kin
+from kin.utils import create_channels
+from sanic import Sanic
+
+from . import log
+from .config import config
+
+VERSION = '1.0.0'
+
+# Setup logging
+log.init_logging()
+logger = logging.getLogger('bootstrap')
+logger.setLevel(config.LOG_LEVEL)
+
+# Setup app-related stuff
+app = Sanic(__name__)
+
+# Setup kin-related stuff
+@app.listener('before_server_start')
+async def setup_kin(app, loop):
+    kin_env = kin.Environment('CUSTOM', config.HORIZON_ENDPOINT, config.NETWORK_PASSPHRASE)
+    app.kin_client = kin.KinClient(kin_env)
+
+    app.minimum_fee = await app.kin_client.get_minimum_fee()
+
+    # Setup channels, skip if we are running in unittests
+    if config.CHANNEL_COUNT > 0 and 'pytest_fixture_setup' not in [a.function for a in inspect.stack()]:
```

Comment:
this "unit test" check is a smell, in case of unit test, you should isolate the logic into some other class and test only it. in case of integration tests (which I think that's the case here) consider, change some external param in tests, to skip this logic, or mock the createChannels call, to return whatever you like under test.